packages/parser/README.md

```diff
@@ -48,7 +48,7 @@ The following additional configuration options are available by specifying them
 
 - **`useJSXTextNode`** - default `true`. Please set `false` if you use this parser on ESLint v4. If this is `false`, the parser creates the AST of JSX texts as the legacy style.
 
-- **`project`** - default `undefined`. This option allows you to provide a path to your project's `tsconfig.json`. **This setting is required if you want to use rules which require type information**. You may want to use this setting in tandem with the `tsconfigRootDir` option below.
+- **`project`** - default `undefined`. This option allows you to provide a path to your project's `tsconfig.json`. **This setting is required if you want to use rules which require type information**. You may want to use this setting in tandem with the `tsconfigRootDir` option below. Note that if this setting is specified, you must only lint files that are included in the projects as defined by the provided `tsconfig.json` files.
```

Comment:
WDYT?

Would it good to suggest that users create a separate `tsconfig.eslint.json` which is essentially

```cjson
{
  "extends": "./tsconfig.json", // path to base tsconfig
  "includes": [
    "src/**/*.ts",
    "test/**/*.ts",
    // etc
  ]
}
```

Reply:
Good idea! Added.